thunder/tests/opinfos.py

```diff
@@ -5931,11 +5931,14 @@ def normalize_sample_generator(op, device, dtype, requires_grad, **kwargs):
         (4, 2, 4, 5),
     )
     for case in cases:
-        yield SampleInput(make(case), eps=1e-8)
-        yield SampleInput(make(case), p=0, eps=1e-8)
-        yield SampleInput(make(case), p=1, eps=1e-8)
-        yield SampleInput(make(case), p=4, eps=1e-8)
-        yield SampleInput(make(case), p=math.inf, eps=1e-8)
+        input_tensor = make(case)
+        # avoid very small norm tensors, which can be unstable to normalize
+        input_tensor = input_tensor + 0.2 * torch.sign(input_tensor)
+        yield SampleInput(input_tensor, eps=1e-8)
```

Comment:
Sample inputs shouldn't reuse the same tensor (which is a subtle property to check for), in case they mutate the tensor. Generate a new tensor each time using the same technique.

Can this use 

https://github.com/k223kim/lightning-thunder/blob/71b4ac9f71a895ef7c5dfe47c23657e2bfb3efb6/thunder/tests/opinfos.py#L66

with zero as a "singularity"?